### PR TITLE
Mybit

### PR DESCRIPTION
**Pull request by walkjivefly**
Please review this, @casey-collab

### CODE REVIEW

File: english/tango/hug.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Hug your partner. Yes, really. Hug them like you love them.
```
Comment (casey-collab) [inline english/tango/hug.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/tango/tango.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Hey, good choice! OK, here's how to tango:
```
Comment (casey-collab) [inline english/tango/tango.md]: This file has also been added to a directory that already contained a page.

File: english/tango/walk.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Walk. One foot in front of the other, preferably in time with the music.
```
Comment (casey-collab) [inline english/tango/walk.md]: This file has also been added to a directory that already contained a page.

File: english/tango/walk/walk.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Walk. One foot in front of the other, preferably in time with the music.
```
Comment (casey-collab) [inline english/tango/walk/walk.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/tango/hug/hug.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Hug your partner. Yes, really. Hug them like you love them.
```
Comment (casey-collab) [inline english/tango/hug/hug.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by walkjivefly** [comment]
I've removed those unreachable leaves. Please review this, @casey-collab

File: english/tango/tango.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+1. Find a partner.
+
+2. Play some music. Golden Age tango would be best, but it's your choice.
+
+3. [Hug](hug.md) your partner.
```
Comment (casey-collab) [inline english/tango/tango.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/tango/tango.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+
+4. [Walk](walk.md) in time with the music.
```
Comment (casey-collab) [inline english/tango/tango.md]: This link is also invalid.

File: english/tango/walk/walk.md (hunk 4 above)
Comment (casey-collab) [inline english/tango/walk/walk.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/tango/hug/hug.md (hunk 5 above)
Comment (casey-collab) [inline english/tango/hug/hug.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by walkjivefly** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**